CONTRIBUTING.rst

```diff
@@ -191,3 +191,17 @@ Done
     Some final work may remain to address review comments; once this is done and the branch is merged the GitHub PR will be closed.
 
 .. _JIRA workflow: https://clusterhq.atlassian.net/
+
+
+.. _reporting-security-issues:
+
+Reporting Security Issues
```

Comment:
In fact definitely do that because "Talk to Us" is the section mentioned in the README.